packages/lib/src/Login.wc.svelte

```diff
@@ -69,20 +78,19 @@
             <Avatar src={userAvatar} {loading} size="sm"></Avatar>
           </slot>
         </button>
-        {#if showPopupMenu}
-          <div
-            class="absolute top-full mt-1.5 right-0 bg-[--s-login-popup-bgcolor] text-[--s-login-popup-textcolor] border-[color:--s-login-popup-border-color] border-[length:--s-login-popup-border-width] rounded-[--s-login-popup-border-radius] w-[--s-login-popup-width] z-[--s-login-popup-zindex]"
-            role="menu"
-          >
-            <slot name="popup-menu">
-              <ul>
-                <LoginMenuItem type="EMAIL" {api} {t} />
-                <LoginMenuItem type="ACCOUNT" {api} {t} />
-                <LoginMenuItem type="LOGOUT" {api} {t} {loading} />
-              </ul>
-            </slot>
-          </div>
-        {/if}
+        <div
+          class={twMerge(
+            'absolute top-full mt-1.5 right-0 bg-[--s-login-popup-bgcolor] text-[--s-login-popup-textcolor] border-[color:--s-login-popup-border-color] border-[length:--s-login-popup-border-width] rounded-[--s-login-popup-border-radius] w-[--s-login-popup-width] z-[--s-login-popup-zindex] hidden',
+            showPopupMenu && 'block'
+          )}
+          role="menu"
+        >
+          <slot name="popup-menu">
+            <LoginMenuItem type="EMAIL" />
```

Comment:
Doesn't this mean that these buttons will be wrapped with Base twice? I can't recall exactly but think there was some issue related to that?

Reply:
We should not have multiple <Base> instances in the same shadow dom, but since we register LoginMenuItem as a custom element it gets its own shadow dom, so we have to load Base in that shadow dom to get base styling etc.

We do the same with <Avatar> above

Reply:
Yeah, that was why I seperated out the renderer so you could use it either directly or as a component

Reply:
Ahh i get it now, i thought it was just a seperation that created extra code, i'll see if i can get it to work like that